src/test/data/script_tests.json

```diff
@@ -168,18 +170,9 @@
 ["1 0 BOOLOR", "NOP", "P2SH,STRICTENC", "OK"],
 ["0 1 BOOLOR", "NOP", "P2SH,STRICTENC", "OK"],
 ["0 0 BOOLOR", "NOT", "P2SH,STRICTENC", "OK"],
-["0x01 0x80", "DUP BOOLOR", "P2SH,STRICTENC", "EVAL_FALSE", "negative-0 negative-0 BOOLOR"],
-["0x01 0x00", "DUP BOOLOR", "P2SH,STRICTENC", "EVAL_FALSE", " non-minimal-0  non-minimal-0 BOOLOR"],
-["0x01 0x81", "DUP BOOLOR", "P2SH,STRICTENC", "OK", "-1 -1 BOOLOR"],
-["0x01 0x80", "DUP BOOLAND", "P2SH,STRICTENC", "EVAL_FALSE", "negative-0 negative-0 BOOLAND"],
-["0x01 0x00", "DUP BOOLAND", "P2SH,STRICTENC", "EVAL_FALSE", " non-minimal-0  non-minimal-0 BOOLAND"],
-["0x01 0x81", "DUP BOOLAND", "P2SH,STRICTENC", "OK", "-1 -1 BOOLAND"],
-["0x01 0x00", "NOT", "P2SH,STRICTENC", "OK", "non-minimal-0 NOT"],
-["0x01 0x80", "NOT", "P2SH,STRICTENC", "OK", "negative-0 NOT"],
-["0x01 0x81", "NOT", "P2SH,STRICTENC", "EVAL_FALSE", "negative 1 NOT"],
-["0x01 0x80 0", "NUMEQUAL", "P2SH", "OK", "-0 0 NUMEQUAL"],
-["0x01 0x00 0", "NUMEQUAL", "P2SH", "OK", "non-minimal-0 0 NUMEQUAL"],
-["0x02 0x0000 0", "NUMEQUAL", "P2SH", "OK", "non-minimal-0 0 NUMEQUAL"],
```

Comment:
For context; there is no non-minimal-0 anymore, so this test makes no sense anymore.